dockerfiles/prod/Dockerfile-nginx

```diff
@@ -0,0 +1,3 @@
+FROM nginx:1.11.5
+COPY contrib/nginx.docker /etc/nginx/conf.d/default.conf
+RUN mkdir -p /opt/jarbas
```

Comment:
This change I agree. The file for dev and prod are exactly the same. I was thinking in move this file to `dockerfiles/Dockerfile-nginx` instead of inside folders dev and prod.